fp_app/spec/factories/schedules.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+    factory :schedule do
+      planner_id { 1 }
```

Comment:
association を持っていて、plannerが存在しないとscheduleは存在しなと思うので以下ようにした方が適切かなと思いました
```suggestion
      planner
```

Reply:
ありがとうございます！
修正しておきます。